plugin/core/windows.py

```diff
@@ -485,10 +488,21 @@ def _handle_message_request(self, params: dict, source: str, client: Client, req
         handler = MessageRequestHandler(self._window.active_view(), client, request_id, params, source)  # type: ignore
         handler.show()
 
+    def restart_session(self, config_name: str) -> None:
+        self._restarting = True
+        self.end_session(config_name=config_name)
+
     def restart_sessions(self) -> None:
         self._restarting = True
         self.end_sessions()
 
+    def end_session(self, config_name: str) -> None:
```

Comment:
Why is `end_config_sessions` not sufficient?

Reply:
And overlook on my part. Will update